pkg/nfs/controllerserver_test.go

```diff
@@ -742,10 +742,10 @@ func TestCopyVolume(t *testing.T) {
 				uuid:    "dst-pv-name",
 			},
 			prepare: func() error {
-				if err := os.MkdirAll("/tmp/snapshot-name/share/snapshot-name/", 0777); err != nil {
+				if err := os.MkdirAll("/tmp/snapshot-name/snapshot-name", 0777); err != nil {
```

Comment:
I should have been more careful when writing unit tests, this case was clearly showing the bug reported by https://github.com/kubernetes-csi/csi-driver-nfs/issues/459 and I just didn't realize it during https://github.com/kubernetes-csi/csi-driver-nfs/pull/430